### PR TITLE
fix: Create workspace as non-admin

### PR DESCRIPTION
**Pull request by gt2345**
## Description

Create workspace as non-admin user throws an error
`Unable to save workspace.
only admin privileged users can set workspace agent user groups`

## Test Plan

Log in as non-admin user
Create a workspace
Should not encounter any error

## Checklist
- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

### CODE REVIEW

File: webui/react/src/hooks/useModal/Workspace/useModalWorkspaceCreate.tsx [hunk 1]
```diff
@@ -227,8 +227,6 @@ const useModalWorkspaceCreate = ({ onClose, workspaceID }: Props = {}): ModalHoo
           checkpointStorageConfig?: unknown;
           name: string;
         } = {
-          agentUserGroup: {},
```
Comment (keita-determined) [inline webui/react/src/hooks/useModal/Workspace/useModalWorkspaceCreate.tsx:230]: is there doc for this usage? its a bit confusing what to set in variables

**Review by keita-determined — APPROVED** [review]
LGTM

File: webui/react/src/hooks/useModal/Workspace/useModalWorkspaceCreate.tsx (hunk 1 above)
Comment (gt2345) [reply webui/react/src/hooks/useModal/Workspace/useModalWorkspaceCreate.tsx:230]: I think our swagger file is the closest thing to API documents...

Comment (keita-determined) [reply webui/react/src/hooks/useModal/Workspace/useModalWorkspaceCreate.tsx:230]: ah ok. the type i put wasnt actually correct. i'll fix it soon

**Pull request merged**